README.md

```diff
@@ -79,10 +79,10 @@ Example resources using verovio
 | [Music Sheet Viewer](http://www.partitionnumerique.com/music-sheet-viewer-wordpress-plugin) | tool | WordPress plugin for displaying graphical music from MEI data | 
 
 
-Digital score repositories on Github
+The Best Digital score repositories on Github
```

Comment:
"Best" is subjective, and this list only provides a few examples and is not comprehensive.